cmd/testwrapper/testwrapper.go

```diff
@@ -1,62 +1,223 @@
 // Copyright (c) Tailscale Inc & AUTHORS
 // SPDX-License-Identifier: BSD-3-Clause
 
-// testwrapper is a wrapper for retrying flaky tests, using the -exec flag of
-// 'go test'. Tests that are flaky can use the 'flakytest' subpackage to mark
-// themselves as flaky and be retried on failure.
+// testwrapper is a wrapper for retrying flaky tests. It is meant to be called
+// in place of 'go test'. It requires the first argument is the pattern to test,
```

Comment:
Can you reword this to not suggest it's a "go test" drop-in replacement? Just say it's its own thing. Maybe "It's an alternative to 'go test' that re-runs tests as needed. It takes different arguments than go test: it requires ..."